frontend/server/src/Controllers/Problem.php

```diff
@@ -4710,6 +4721,10 @@ public static function getProblemDetailsForTypeScript(
 
         $response['templateProperties']['payload']['problem']['nextSubmissionTimestamp'] = $nextSubmissionTimestamp;
 
+        if ($isQualityReviewer) {
+            $response['templateProperties']['payload']['alreadyReviewedPayload'] = $alreadyReviewedPayload;
```

Comment:
Estamos intentando que los elementos en las anotaciones puedan ser `null` o en su defecto opcionales, pero evitar el declararlos con ambas anotaciones, para eso se me ocurre que aquí sólo lo asignemos cuando el usuario sea reviewer y además de que ya haya sido revisado, sería algo así:
```suggestion
        if ($isQualityReviewer && $alreadyReviewedPayload) {
            $response['templateProperties']['payload']['alreadyReviewedPayload'] = $alreadyReviewedPayload;
```
de esta forma ya podrías quitar el `|null` en la declaración de `alreadyReviewedPayload` en L43